grunt-configs/shell.js

```diff
@@ -24,12 +24,12 @@ module.exports = function(grunt, options) {
                      'static/abtests.json'
         },
 
-        npmInstall: {
-            command: 'npm prune && npm install'
+        npmInstallFaciaTool: {
+            command: 'cd facia-tool/public && npm prune && npm install'
         },
 
         jspmInstallFaciaTool: {
-            command: 'node ../../node_modules/jspm/jspm.js install',
+            command: './node_modules/.bin/jspm install',
```

Comment:
I was reluctant to touch the install here incase @piuccio had it set up in a way he wanted, but if you're in there, might be worth replicating the `./jspm install && ./jspm dl-loader && ./jspm clean` functionality? keep it clean etc. just an idea, nothing more